src/toolbelt/core.cljc

```diff
@@ -151,20 +153,42 @@
   elements of xs return the same value under f, the first is returned"
   [f xs]
   (let [s (atom #{})]
-    (for [x     xs
-          :let  [id (f x)]
+    (for [x xs
+          :let [id (f x)]
           :when (not (contains? @s id))]
       (do (swap! s conj id)
           x))))
 
 
-(defn round
-  [x & [precision]]
-  (if (some? precision)
-    (let [scale (Math/pow 10 precision)]
-      (-> x (* scale) Math/round (/ scale)))
-    (Math/round x)))
+#?(:clj
+   (defn round
+     [x & [precision]]
+     (if (some? precision)
+       (let [scale (Math/pow 10 precision)]
+         (-> x (* scale) Math/round (/ scale)))
+       (Math/round x)))
+
+   :cljs
+   (defn round
+     [x & [precision]]
+     (if (some? precision)
+       (.parseFloat (.toFixed x precision))
+       (.round js/Math x))))
+
+
+#?(:clj
+   (defn throwable? [x]
+     (instance? java.lang.Throwable x))
+
+   :cljs
+   (defn throwable? [x]
+     (instance? js/Error x)))
+
+
+(defn all-some?
+  "Given seq as xseq returns true if all its items are not nils, otherwise false"
```

Comment:
Also returns `false` on empty collection, would that be expected? I'm not sure where I stand on that, but perhaps add into the doc string to clarify that at least.

Reply:
No, it shouldn't. Investigating...

Reply:
Sorry, it definitely should return nil on empty connection because it returns true only if collection contains any non-nil value.